modules/View.js

```diff
@@ -56,6 +56,72 @@
       if (!options.noActivate) {
         this.activate();
       }
+      // this.updateDelegateEvents();
+    },
+
+
+    /**
+    * called by updateDelegateEvents to wrap callback functions of event calls
+    * with event triggers
+    * @param method {function} callback method of event
+    * @param eventName {String} description of event type 
+    * @return modified mehtod with before/after event triggers
+    * @method trackEvents
+    */
+    trackEvents : function(method, eventName){
+      var self = this;
+      var methodCopy = method;
+      method = _.bind(function(){
+        var uuid = (new Date()).getTime().toString(16)+Math.floor(1E7*Math.random()).toString(16);
+        this.trigger('clickEventTiming', {
+          uuid: uuid,
+          type: 'clickEvent',
+          state: 'start',
+          eventName: eventName,
+          time: Date.now(),
+        });
+
+        methodCopy.call(self);
+
+        this.trigger('routetiming', {
+          uuid: uuid,
+          state:'end',
+          time: Date.now(),
+        });
+      },this);
+      return method;
+      },     
+
+    /**
+    * updates Backbone's delegateEvents to bind triggers to all event callback functions 
+    * Binds DOM events with the view using events hash.
+    * Also adds feedback event bindings
+    * functions in the View's list of events
+    * @method updateDelegateEvents
+    * @override
+    */
+    delegateEvents: function(events){
+      var delegateEventSplitter = /^(\S+)\s*(.*)$/;
+      if (!(events||(events = _.result(this,'events')))) return this;
+      var eventsClone = _.extend({},events);
+      for (var key in eventsClone) {
+        var method = eventsClone[key];
+        if (!_.isFunction(method)) method = this[eventsClone[key]];
+        if (!method) continue;      
+        this.trackEvents = _.bind(this.trackEvents, this);
+        var methodNew = this.trackEvents(method, '');
+        eventsClone[key] = methodNew;
+      }
+
+      Backbone.View.prototype.delegateEvents.call(this, eventsClone);
```

Comment:
Nice. You're capturing it before the delegateEvents is fired and updating the parameter.